docs/b2b-edition/docs/buyer-portal/overview.mdx

```diff
@@ -0,0 +1,98 @@
+- **Headless (self-hosted):** Run and deploy the open-source Buyer Portal next to your headless storefront. See the [Headless guide](/b2b-edition/docs/buyer-portal//headless).
+- **Catalyst:** Render the Buyer Portal on top of Catalyst and wire authentication. See the [Catalyst guide](/storefront/catalyst/experiments/buyer-portal).
+
+## Prerequisites
+
+- **Node.js**
+  - We recommend managing installations with [nvm](https://github.com/nvm-sh/nvm).
+- **Yarn package manager**
+  - Install globally with `npm i -g yarn`, or enable locally using `corepack enable yarn` to ensure the correct project version.
+- **Git**
+  - Required for cloning repositories and managing code.
+- **BigCommerce store with B2B Edition enabled**
+  - Needed for configuring and deploying the Buyer Portal.
+  - A sandbox environment with API access (optional, but useful for live stores)
+  - At least one storefront with B2B Edition enabled from B2B > Storefronts.
+- **React and JavaScript/TypeScript experience**
+- **V3 BigCommerce API token with B2B Edition scope** (for advanced integrations)
+
+See the [B2B Buyer Portal README](https://github.com/bigcommerce/b2b-buyer-portal) for the current minimum required versions of Node and Yarn.
+
+### Enabling B2B Edition for a storefront
+
+1. In the B2B Edition dashboard, navigate to Storefronts and click the ellipsis (...) next to the relevant storefront channel. Click “Enable B2B” and ensure the channel updates to “B2B Enabled.”
+   ![Enable Buyer Portal](https://storage.googleapis.com/bigcommerce-production-dev-center/images/b2b/enable-buyer-portal.png)
+2. Also in the Storefronts section of the dashboard, navigate to “Edit settings” for the storefront, select “Buyer portal” settings, and verify the “Buyer portal type” matches your intended configuration.
+   ![Buyer Portal Type - Catalyst - 1](https://storage.googleapis.com/bigcommerce-production-dev-center/images/b2b/buyer-portal-type-catalyst-1.png)  
+   ![Buyer Portal Type - Catalyst - 2](https://storage.googleapis.com/bigcommerce-production-dev-center/images/b2b/buyer-portal-type-msf-2.png)  
+   ![Buyer Portal Type - Catalyst - 3](https://storage.googleapis.com/bigcommerce-production-dev-center/images/b2b/buyer-portal-type-msf-3.png)
+
+<Callout type='info'>
+The default version of Buyer Portal may be used on any storefront type; however, configuration steps will vary between types.
+</Callout>
+
+## Buyer Portal Concepts
+
+### What is the Buyer Portal?
+
+The Buyer Portal is a React application providing B2B buyers a dedicated interface for business-specific workflows in BigCommerce. It connects to the B2B GraphQL API and supports two main configurations:
+
+- **Hosted Buyer Portal:** Managed by BigCommerce, updated automatically.
+- **Custom Buyer Portal:** Forked and customized by merchants/agencies for advanced branding, features, or integrations.
+
+### Architecture (high level)
```

Comment:
My main piece of feedback would be to modify this section.  Instead of the current bullet points, my recommendation would be to list the key parts of the tech stack:

- The B2B GraphQL API (link to our docs)
- React (https://react.dev/)
- React Router (https://reactrouter.com/)
- Material UI (https://mui.com/material-ui/)
- Redux (https://redux.js.org/)